### PR TITLE
Change layout

### PR DESCRIPTION
**Pull request by yevbar**
The site now has two sections where the top section holds events that have not occurred whereas the bottom second holds events that have already occurred

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default class extends Component {
       searchLat: null || props.searchLat,
       searchLng: null || props.searchLng,
       formattedAddress: undefined,
-      timeFilter: 'school year',
+      timeFilter: 'bazinga',
```
Comment (maxwofford) [inline src/pages/index.js]: What does this do? It looks like you already removed all uses of `this.state.timeFilter`

File: src/pages/index.js [hunk 2]
```diff
@@ -237,10 +221,13 @@ export default class extends Component {
               </Link>
               {sortByProximity && formattedAddress && ` to ${formattedAddress}`}.
             </Text>
+            <Text color="muted" mt={4} mb={3}>
+              Events that are coming right up&#33;
```
Comment (maxwofford) [inline src/pages/index.js]: Is there a reason we're using `&#33;` here?

File: src/pages/index.js [hunk 3]
```diff
@@ -264,6 +251,36 @@ export default class extends Component {
                   />
                 ))}
             </Flex>
+            <Container maxWidth={36} px={3} align="center">
+              <Text color="muted" mt={4} mb={3}>
+                Events that have already passed&#33;
```
Comment (maxwofford) [inline src/pages/index.js]: `&#33;`

Comment (maxwofford) [reply src/pages/index.js]: I think this could actually just drop the exclamation & be fine

File: src/pages/index.js [hunk 4]
```diff
@@ -264,6 +251,36 @@ export default class extends Component {
...
+              </Text>
+            </Container>
+            <Flex mx={[1, 2, -3]} wrap justify="center">
```
Comment (maxwofford) [inline src/pages/index.js]: Now that there are 2 of these on the page, want to make something like an `EventsList` component?

**Review by maxwofford — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 1 above)
Comment (yevbar) [reply src/pages/index.js]: That was put in as a placeholder filter

File: src/pages/index.js (hunk 2 above)
Comment (yevbar) [reply src/pages/index.js]: My editor complained about the exclamation mark, changing

File: src/pages/index.js (hunk 3 above)
Comment (yevbar) [reply src/pages/index.js]: Alright, leaving exclamation mark out

**Comment by yevbar** [comment]
The git inserts and notes got in :\

**Comment by lachlanjc** [comment]
I'd like to implement these changes. Can you move the good commits to a new branch and PR? Thanks :)

**Pull request closed without merging**